src/Form/Form.js

```diff
@@ -325,9 +328,7 @@ class Form {
 
         this.isAutofilling = false
 
-        if (this.tooltip) {
```

Comment:
I removed these as we're checking the value there anyway and it's a method call to device interface now.